src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,69 @@
+package frc.robot.subsystems;
+
+import java.util.function.BooleanSupplier;
+
+import com.chopshop166.chopshoplib.commands.SmartSubsystemBase;
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.maps.RobotMap.TelescopeMap;
+
+public class Climber extends SmartSubsystemBase {
```

Comment:
We should probably have a command that controls the elevator with the triggers so we can try different speeds easily.

Reply:
This exists now so I'll resolve this